tools/check_tidy.py

```diff
@@ -28,6 +30,16 @@
 from common_py.system.executor import Executor as ex
 
 
+def parse_option():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--autoedit', action='store_true', default=False,
+        help='edits to files instead of displaying a diff' +
```

Comment:
There is no need for the `+` sign. The concatenation will also happen without it (strings are automatically concatenated).